src/views/ops/Template/components/Variable.vue

```diff
@@ -81,7 +81,7 @@ export default {
         if (oldVal === undefined) return
         if (newVal.length > 0 || !this.initial) {
           newVal.map((item) => {
-            item.default_value = item.text_default_value || item.select_default_value
+            item.default_value = item.text_default_value || item.select_default_value || undefined
           })
           this.$emit('input', newVal)
         }
```

Comment:
The above code does not include any regular expressions or patterns to detect irregularities, potential issues, or optimize the script. Could you please specify some areas where you need checks?